### DISCUSSION TITLE
How to view macro to macro and macro to pin connections in openroad gui mode?

### DISCUSSION THREAD

**Discussion by khaledhasanzami** [The-OpenROAD-Project/OpenROAD#3030] [General]
I am having some error showing macros can not connect to some pins. 

I can't figure out how to view macro to macro connections (flightline) and macro to pin connections in gui mode. 

Help me if anyone figured this out!

**Comment by vijayank88** [comment]
Use `net` name to view in GUI using `Find` menu. It was explained here: https://openroad-flow-scripts.readthedocs.io/en/latest/tutorials/FlowTutorial.html#tracing-the-clock-tree

**Comment by khaledhasanzami** [comment]
I was looking for macro to macro connection. 

I got the solution though. 

We have to select a macro, right click, highlight, go to all nets and chose a group.

Thank you for the suggestions though.
